spec/factories/item.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :item do
+    type              { %w[HodinkeeItem CrownandcaliberItem BobswatchesItem].sample }
```

Comment:
try this https://apidock.com/rails/Class/descendants

Reply:
In my case It returns empty array:
![Screenshot from 2022-03-29 09-31-01](https://user-images.githubusercontent.com/94611266/160557989-bd5ebde5-a447-4cbd-bb19-a48eb1f7c2fa.png)